docs/topic-guides/elastic-infra/aws-spot.txt

```diff
@@ -0,0 +1,74 @@
+.. _aws-spot:
+
+####################
+ AWS Spot Instances
+####################
+
+This document describes how AWS spot instances work. Determined handles
+most details of working with spot instances, but knowing how spot
+instances work at a high level is helpful for understanding the behavior
+of Determined when provisioning spot instances.
+
+*******************
+ On-Demand vs Spot
+*******************
+
+The standard way to create an EC2 instance is to create an on-demand
+instance. On-demand instances always have the same price and once you
+have created one, you are guaranteed to have that instance forever
+(barring hardware failure).
+
+The number of EC2 instances desired by customers varies over time. AWS
+wants to have enough hardware so that during peak usage periods, all
+customers are able to get as many on-demand instances as they need. This
+means that during non-peak periods (which is most of the time), AWS has
+extra hardware capacity sitting around unused. AWS makes these extra
+instances available to customers via the spot market as spot instances.
+They can be rented at a much reduced cost, but the trade-off is that, if
+AWS needs instances to satisfy on-demand users, they can reclaim your
+spot instance and give it to the on-demand user. If all available
+instances are in use by on-demand customers, you will not be able to
+launch any spot instances.
+
+********************
+ Spot On Determined
+********************
+
+Because they can be reclaimed, using spot instances requires you to have
+good fault tolerance built into your software. Determined was built with
+fault-tolerance as a core feature, so using spot instances is usually as
+easy as setting ``spot: true`` in the master configuration.
+
+AWS might not always have the capacity to fulfill spot requests. When
+AWS is out of capacity, the Determined cluster will wait until AWS has
+capacity and can fulfill the requests. This will be visible in the
+master logs.
+
+**************
+ Spot Pricing
+**************
+
+Unlike on-demand instances, the market price of a spot instance varies.
+Once the spot instance has been created, the hourly price is constant,
```

Comment:
"the hourly price is constant" -> "the hourly cost to run that instance is constant"